Demo/DemoTests/MoyaProviderSpec.swift

```diff
@@ -143,7 +139,7 @@ class MoyaProviderSpec: QuickSpec {
                             executed = true
                             return endpoint.urlRequest
                         }
-                        provider = MoyaProvider(endpointsClosure: endpointsClosure, endpointResolver: endpointResolution, stubResponses: true)
+                        provider = MoyaProvider<GitHub>(endpointResolver: endpointResolution, stubBehavior: MoyaProvider.ImmediateStubbingBehaviour)
```

Comment:
`endpointsClosure` vs `endpointResolver` caught my eye ("endpoints" vs "endpoint"), are these "similar-yet-different"?  I'm not familiar w/ the resolver.